packages/functional-tests/src/tests/asset-preload-test.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
+import App from '../app';
+import delay from '../utils/delay';
+import destroyActors from '../utils/destroyActors';
+import Test from './test';
+
+export default class AssetPreloadTest extends Test {
```

Comment:
How is this a new file? This is an old test.

Reply:
Updated the file name, since it was the only one that didn't end in `-test`. It seems someone else did the same.